tools/abel/README.md

```diff
@@ -0,0 +1,119 @@
+# ABEL
+### Automated Building Entity Loader
+
+## Overview
+ABEL is a tool allowing system integrators to easily and efficiently create and modify [Building Configuration files](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md).
+
+## Setup and installation
+
+### Setting up a tooling environment
+
+Clone the Digital Buildings respository
+
+  ```
+  git clone https://github.com/google/digitalbuildings.git
+  ```
+
+Navigate to the Digital Buildings Repository's tooling library
+
+  ```
+  cd digitalbuildings/tools/
+  ```
+
+Create a virtual environment with an environment name, in this example: `tooling`
+
+  ```
+  virtualenv tooling
+  ```
+
+Activate your virtual environment
+
+* MacOs/Linux:
+
+  ```
+  source tooling/bin/activate
+  ```
+
+* Windows:
+
+  ```
+  tooling/Scripts/activate
+  ```
+
+Depending on your operating system run either of the global setup scripts to configure dependencies
+
+* Linux/MacOs:
+  ```
+  bash pip_install.sh
+  ```
+
+* Windows:
+
+  ```
+  pip_install.bat
+  ```
+
+### Obtain a spreadsheet token for your google sheets account
+
+1. Be added to a GCP service account with `Service Account User` and `Service Account Token Creator` roles.
+
+    * Please contact your IoT TPM for details and to be added to a project.
+
+2. Create a token request file, `token_request.json` with contents:
+
+    ```
+    {"scope": [    "https://www.googleapis.com/auth/spreadsheets" ],  "lifetime": "3600s"}
+    ```
+
+3. Run the below curl command to obtain a token:
+
+    ```
+        curl -X POST \
+        -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) \
+        -H "Content-Type: application/json; charset=utf-8" \
+        -d @token_request.json \
+        "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/your-service-account@project-id.iam.gserviceaccount.com:generateAccessToken" >| spreadsheet_token.json
+    ```
+
+4. `spreadsheet_token.json` should look something like:
+
+    ```
+    {
+      "accessToken": "ya29.c.AXv0zTOa9p5K78GtGntoIQCqyt2R572qZ3…",
+      "expireTime": "2022-02-24T19:57:26Z"
+    }
+    ```
+
+## The ABEL Spreadsheet
+The ABEL spreadsheet serves as a frontend for ABEL.
+
+Please see the [ABEL spreadsheet docs](./validators/README.md) for detailed instructions on how to use the spreadsheet.
```

Comment:
this should be avoided. please follow similar path here: https://github.com/google/digitalbuildings/commit/a21b6d71829e06862a2427d3c4dbe44e621ad6e5